styles.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+#back-images-intro {
+  width: 100%;
+  height: 100%;
+  padding-top: 24px;
+  background-image: url('./images/header-shapes\ mobile.svg');
+}
+
+.connectlink {
+  padding: 12px;
+  font-size: 16px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: left;
+  color: #7f8cff;
+}
+
+h1 {
+  padding-left: 12px;
+  width: 327px;
+  height: 150px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  padding-top: 206px;
+  color: #172b4d;
+}
+
+header {
+  background: #e5e5e5;
+  box-sizing: border-box;
+  left: 0;
+  padding: 24px;
+  position: fixed;
+  top: 0;
+  z-index: 1000;
+  width: 100%;
+}
+
+html {
+  font-family: 'Poppins', serif;
+  font-size: 16px;
+}
+
+.intro {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.intro-text {
+  width: 327px;
+  height: 146px;
+  padding: 12px;
+  box-sizing: border-box;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #344563;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  border-radius: nullpx;
+}
+
+.logonav {
+  background: #e5e5e5;
+  list-style-type: none;
+}
+
+main {
+  background-image: url("./images/header-background-mobile.svg");
+  background-repeat: no-repeat;
```

Comment:
- [ ] [OPTIONAL] The background is repeating, please consider fixing this so your design is like the Figma design

|Your design|Figma design|
|---|---|
|![Screenshot_20220526_180657](https://user-images.githubusercontent.com/25227022/170539230-01fd2d7d-0277-4cf1-aa33-fa9319e2c6b5.png)|![Screenshot_20220526_181032](https://user-images.githubusercontent.com/25227022/170539407-9ff5a7eb-0274-45ee-bbd9-dcbe7752c4d1.png)|